frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -71,14 +72,26 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample }) => {
         required
       />
       {values.git.showGitType && (
-        <DropdownField
-          name="git.type"
-          label="Git Type"
-          items={GitTypes}
-          title={GitTypes[values.git.type]}
-          fullWidth
-          required
-        />
+        <>
+          <DropdownField
+            name="git.type"
+            label="Git Type"
+            items={GitReadableTypes}
+            title={GitReadableTypes[values.git.type]}
+            fullWidth
+            required
+          />
+          {!gitTypeTouched && (
+            <Alert
```

Comment:
We should use the help text or error field? not sure a separate alert is correct.

Reply:
Unless I am mistaken, the error text won't allow the form to submit. As the field has an error.

Discussed with Serena and the info text was swapped out for the Alert as "typically we don't show informative info under the dropdown".

Reply:
Thanks for the clarification.